hack/remove-users-without-contributions.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+if [[ "$1" =~ -h ]]; then
+    cat << EOF
+usage: $0 <input-file>
+
+    Removes all users without contributions (taken from the input file) from the orgs.yaml
+
+    input file in csv format is manually fetched from KubeVirt devstats:
+    https://kubevirt.devstats.cncf.io/d/48/users-statistics-by-repository-group?orgId=1&from=now-1y&to=now&var-period=w&var-metric=activity&var-repogroup_name=All&var-users=All
+
+    then from the top panel select "Inspect" > "Data" and select "Download CSV"
+EOF
+    exit 0
+fi
+
+if [ ! -f "$1" ]; then
+    echo "file $1 doesn't exist"
+    exit 1
+fi
+input_csv_file="$1"
+
+# generate input file from csv where each user is in a separate line
+head -1 "$input_csv_file" \
+    | tr ',' "\n" \
+    | sed 's/"//g' \
+    | sort -u \
+    > /tmp/users-with-contributions.txt
+
+# add bot, admin and users with invisible contributions (i.e. OSPO etc)
+cat << EOF >> /tmp/users-with-contributions.txt
```

Comment:
I understand the bots exception, but I am not sure I understand the other members exceptions.
Maybe different members have different reasons. If we do not specify it now, it will be hard to know later the reason.

Reply:
@EdDev good point! I'll try to add those in separate steps with an explanation.